apps/fabric-website/index.html

```diff
@@ -30,7 +30,7 @@
 <body>
   <div id="main">
     <div class="loading">
-      <img class="loadingImage" src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/loading.gif" alt="Loading"
+      <img class="loadingImage" src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/spinner.gif" alt="Loading"
```

Comment:
This is not a blocking comment but, i beleive we could use the office-ui-fabric-react spinner. If that makes this more complicated then let's just merge it in.

Reply:
@battletoilet I initially looked into this issue with the goal of using the office-ui-fabric-react spinner, but then discovered that part of why the loading gif is there is to spin while we're loading dependencies like react. Confirmed these suspicions with @mikewheaton and @Jahnp so we got Colton to replicate our spinner in a gif.